package.json

```diff
@@ -0,0 +1,79 @@
+{
+  "name": "cards-exporter",
+  "version": "0.0.1",
+  "description": "Cards exporter for 'Memory Cards' project",
+  "main": "index.js",
+  "engines": {
+    "node": ">=10.0.0",
+    "yarn": ">=1.3.0"
+  },
+  "scripts": {
+    "dev": "nodemon -e ts,js --exec 'yarn start'",
+    "heroku-postbuild": "yarn run start:build",
+    "lint:prettier": "prettier --list-different --parser typescript ./pages/**.{ts,tsx}",
+    "lint:tslint": "tslint --project .",
+    "lint": "yarn lint:prettier && yarn lint:tslint",
+    "precommit": "lint-staged",
+    "prettier": "prettier --parser typescript --write ./**/*.tsx",
+    "start:build": "NODE_ENV=production next build",
+    "start:server": "NODE_ENV=production ts-node server.ts",
+    "start": "yarn start:build && yarn start:server",
+    "test:jest": "jest",
+    "test": "yarn lint && yarn test:jest"
+  },
+  "repository": {
+    "type": "git",
+    "url": "git+https://github.com/memory-cards/cards-exporter.git"
+  },
+  "keywords": [
+    "anki"
+  ],
+  "lint-staged": {
+    "*.{ts,tsx}": [
+      "prettier --parser typescript --write",
+      "tslint --fix -p .",
+      "git add"
+    ],
+    "*.{js,json}": [
+      "prettier --write",
+      "git add"
+    ]
+  },
+  "author": "@vvscode",
+  "license": "ISC",
+  "bugs": {
+    "url": "https://github.com/memory-cards/cards-exporter/issues"
+  },
+  "homepage": "https://github.com/memory-cards/cards-exporter#readme",
+  "dependencies": {
+    "@zeit/next-typescript": "^1.1.1",
+    "enzyme": "^3.7.0",
+    "express": "^4.16.4",
+    "fork-ts-checker-webpack-plugin": "^0.4.14",
+    "next": "^7.0.2",
+    "node-mocks-http": "^1.7.3",
+    "react": "^16.6.0",
+    "react-dom": "^16.6.0",
+    "react-test-renderer": "^16.6.0",
+    "tslint": "^5.11.0",
+    "typescript": "^3.1.3"
+  },
+  "devDependencies": {
+    "@types/enzyme": "^3.1.14",
+    "@types/express": "^4.16.0",
+    "@types/jest": "^23.3.8",
+    "@types/next": "^7.0.3",
+    "@types/react": "^16.4.18",
+    "@types/react-test-renderer": "^16.0.3",
+    "enzyme-adapter-react-16": "^1.6.0",
+    "husky": "^1.1.2",
```

Comment:
Vasiliy, do we have a setup for this library? Or it will be added later?

Reply:
it used for https://github.com/memory-cards/cards-exporter/pull/1/files/bea9a9410ae615582d0674241d264956e03a4849#diff-b9cfc7f2cdf78a7f4b91a753d10865a2R16

it add precommit hook

Reply:
I've found this dependency, thanks.